README.md

```diff
@@ -29,15 +29,21 @@ There is also support for legacy XMPP protocol for FCM.
         
         This will start the database at `localhost:9001`
         
-   3.2. To use as an embedded in-memory database instance (Not recommended for production deployments), set the `spring.datasource.url=jdbc:hsqldb:mem:/appserver` in `application-<profile>.properties`.
+   3.2. To use as an embedded in-memory database instance (Not recommended for production deployments), set the `spring.datasource.url=jdbc:hsqldb:mem:/appserver` in `application-<profile>.properties`.  Also, change the properties in `src/main/resources/application.properties` to dev or prod according to your requirements. 
 
-4. Build the project using gradle wrapper and run using spring boot. Note: This project uses JAVA 11, please download and install it before building. On mac or linux, run the below -
+4. Build the project using gradle wrapper and run using spring boot. Note: This project uses JAVA 11, please download and install it before building. 
+
+5. The build will need to create a logs directory. The default path is `/usr/local/var/lib/radar/appserver/logs`. Either create the directory there using `sudo mkdir -p /usr/local/var/lib/radar/appserver/logs` followed by `sudo chown $USER /usr/local/var/lib/radar/appserver/logs` or change logs file directory in `src/main/resources/logback-spring.xml` to local log directory like `<property name="LOGS" value="logs" />`
+
+6. You need to configure adminSDK now. Follow the official Firebase [documentation](https://firebase.google.com/docs/admin/setup#initialize-sdk) till you setup the enviroment variable. 
```

Comment:
It would be better here to specify -
```markdown
6. There are 2 ways to communicate with the Firebase Cloud Messaging- XMPP and Admin SDK. At least one of these needs to be configured. To configure these, please look at the [FCM section](#fcm).
```

Then divide the FCM section below into two sub-headings- XMPP and AdminSDK. The XMPP config is already mentioned there. You can also specify which sender config to use for each one and setting xmpp upstream enabled to false when using Admin SDK.

Reply:
Okay, will do that in the next commit.

Reply:
Hey, @yatharthranjan, I have made the mentioned changes in README. Let me know if you think any other changes are needed.